repose-aggregator/components/filters/keystone-v2/src/main/resources/META-INF/schema/config/keystone-v2.xsd

```diff
@@ -0,0 +1,383 @@
+
+    <xs:complexType name="KeystoneV2Config">
+        <xs:annotation>
+            <xs:documentation>
+                <html:p>The root config type for the Rackspace Keystone V2 filter configuration file.</html:p>
+            </xs:documentation>
+        </xs:annotation>
+
+        <xs:all>
+            <xs:element name="identity-service" type="IdentityServiceType" minOccurs="1" maxOccurs="1"/>
+            <xs:element name="delegating" type="DelegatingType" minOccurs="0" maxOccurs="1"/>
+            <xs:element name="white-list" type="WhiteListType" minOccurs="0" maxOccurs="1"/>
+            <xs:element name="cache" type="CacheType" minOccurs="0" maxOccurs="1"/>
+            <xs:element name="tenant-handling" type="TenantHandlingType" minOccurs="0" maxOccurs="1"/>
+            <xs:element name="require-service-endpoint" type="ServiceEndpointType" minOccurs="0" maxOccurs="1"/>
+        </xs:all>
+    </xs:complexType>
+
+    <xs:complexType name="WhiteListType">
+        <xs:annotation>
+            <xs:documentation>
+                <html:p>List of URI Regular expression patterns to bypass authentication and authorization</html:p>
+            </xs:documentation>
+        </xs:annotation>
+        <xs:sequence>
+            <xs:element name="uri-regex" type="xs:string" minOccurs="1" maxOccurs="unbounded"/>
+        </xs:sequence>
+    </xs:complexType>
+
+    <xs:complexType name="IdentityServiceType">
+        <xs:attribute name="username" type="xs:string" use="required">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>Admin username to access Rackspace Keystone V2 identity service.</html:p>
+                </xs:documentation>
+            </xs:annotation>
+
+        </xs:attribute>
+        <xs:attribute name="password" type="xs:string" use="required">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>Admin user password to access Rackspace Keystone V2 identity service.</html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+
+        <xs:attribute name="uri" type="xs:anyURI" use="required">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>Rackspace Keystone v2 Identity Endpoint URI</html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+
+        <!-- todo: the AkkaServiceClient uses the default pool and is hard-coded... -->
+        <!--xs:attribute name="connection-pool-id" type="xs:string" use="optional">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>Http Connection pool ID to use when talking to Rackspace Keystone V2 Identity</html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute-->
+        <xs:attribute name="set-roles-in-header" type="xs:boolean" use="optional" default="true">
```

Comment:
@dkowis wrote them in when he wrote the config, and I didn't have the heart to remove them.

Actually, they just seem like things we may want in the future so I left them.